### DISCUSSION TITLE
Give an option to show the entire secret key while activating 2FA on an account

### DISCUSSION THREAD

**Discussion by DecemberGiraffe** [standardnotes/forum#3263] [Suggestions]
**Describe the bug**
On android, there are two buttons (one for copy and the other I cannot recall) that overlap a portion of the secret key preventing the user from seeing the entire key. This issue appears on two consecutive screens during the 2FA setup process.

**To Reproduce**
1. Go to Preferences then Security.
2. Enable 2FA
4. Next screen shows secret key but can only see the first half or two thirds with the rest blocked by buttons.
5. Going to the next screen also shows the same issue. The secret key is shown again but a portion is blocked from being visible.

**Expected behavior**
View entire secret key with the buttons to the right of the field with the key.

**Smartphone (please complete the following information):**
 - Device: Pixel 6a
 - OS: Android 13
 - Version 3.167.27

**Additional context**

**Comment by effieeee** [comment]
Hi @DecemberGiraffe, you can actually tap on the **copy** button to copy the entire key, though is there any particular reason that you need to see the entire key in the box right away?

**Comment by DecemberGiraffe** [comment]
For security reasons I prefer to type it in manually

**Comment by effieeee** [comment]
I see, thank you for clarifying! I’ve renamed this ticket and placed it as a suggestion under the Discussions page for now 🙏
